lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -134,16 +120,6 @@ public Map<String, Object> handleRequest(JourneyRequest event, Context context)
                     gpg45ProfileEvaluator.parseCredentials(
                             userIdentityService.getUserIssuedCredentials(userId));
 
-            if (!checkCorrelation(userId)) {
-                return JOURNEY_PYI_NO_MATCH.toObjectMap();
-            }
```

Comment:
I'm not sure we should remove this check - it feels quite good to double-check correlation at the end as well (as it's part of the GPG45 algorithm), what do you think?

(We should be able to use the commonised checking logic in `UserIdentityService` though)

Reply:
Ok, that makes sense!